SECP-service/src/main/java/com/visucius/secp/resources/GroupResource.java

```diff
@@ -0,0 +1,32 @@
+package com.visucius.secp.resources;
+
+
+import com.codahale.metrics.annotation.Timed;
+import com.visucius.secp.Controllers.GroupController;
+import com.visucius.secp.DTO.GroupCreationRequest;
+import io.dropwizard.auth.Auth;
+import io.dropwizard.hibernate.UnitOfWork;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+@Path("/")
```

Comment:
This should be /group. In addition. Can you add the @RolesAllowed("ADMIN") annotation. We don't want anyone to have access to this resource.